### DISCUSSION TITLE
Render API & Telnyx Messaging Profile Issue

### DISCUSSION THREAD

**Discussion by gh4E1NtQii** [0perationPrivacy/VoIP#152] [Q&A] [answered]
### Issue
After adding a number in VoIP Suite application and when prompted to Override Settings I clicked Yes.  I noticed I was able to send text messages from VoIP Suite but could not receive them.

Log into Telnyx and checked the Messaging profile that gets added to the phone number which I just associated with VoIP Suite.  

1. Login To Telnyx
2. Click `Messaging` on the left hand side
3. Under `Configure your Messaging Profiles` section find `VoIP sms Web Application`  - There will be multiple if you have multiple Telnyx phone numbers.
4. Click the `pencil icon`
5. Scroll down to the `Inbound Settings` section
6. Under the `Send a webhook to this URL:` section you will see the URL for receiving sms text messages via the Render API.  This is incorrect.

It is missing a forward slash in-between `.com` and `api`

### Resolution

**Change From**
     Send a webhook to this URL:  https://voipsuite-xxxx.onrender.comapi/setting/receive-sms/telnyx

**Change To**
     Send a webhook to this URL:  https://voipsuite-xxxx.onrender.com/api/setting/receive-sms/telnyx

Once I made the above change, I was able to receive text messages to my Telnyx number on VoIP Suite

Anyone else seeing this in their experience?

**Comment by 0perationPrivacy** [comment, marked as answer]
I could replicate your issue.
It happens when you don't put a leading `/` in the environment variable `BASE_URL` in Heroku or Render.
So fix it, please make sure you have slash at the end of your url in Render under BASE_URL like https://subdomain.domain.com/

I will make sure I highlight this in the documentation and also fix it in the next update to add it automatically if someone forgets.
Thanks

**Reply by gh4E1NtQii** [reply]
Ah, okay.  I was using the instructions on [IntelTechniques](https://inteltechniques.com/voip.suite.html) 

I will use the GitHub Wiki and the Extreme Privacy book as references going forward.  My apologies


### Instructions on IntelTechniques
Click "Create Web Service"
Save the URL such as "https://voip-14.onrender.com"
Click "Environment"
Click "Add Environment Value"
Enter "BASE_URL" in the "KEY" field
Enter the saved URL in the "VALUE" field
Click "Save Changes"

**Reply by 0perationPrivacy** [reply]
IntelTechniques page is now updated. Thanks for pointing it out 👍🏻
